btcec/schnorr/musig2/sign.go

```diff
@@ -606,6 +635,8 @@ type combineOptions struct {
 	combinedKey *btcec.PublicKey
 
 	tweakAcc *btcec.ModNScalar
+
+	adaptorPub *btcec.JacobianPoint
```

Comment:
Where is this actually used? It seems like you can set it with `WithAdaptorCombine`, but don't see it being read anywhere. Which indicates to me that we probably want a unit test case for this as well.

Reply:
You're right this is unnecessary. I've removed it along with `WithAdaptorCombine`.